### PR TITLE
New beaker image for evaluating OLMo-1.7 on Open-Instruct eval suite

### PR DESCRIPTION
**Pull request by nouhadziri**
updated the docker file & Added the new beaker image to README

### CODE REVIEW

**Comment by natolambert** [comment]
This seems fine, why pin setuptools < 70 @nouhadziri?

**Comment by nouhadziri** [comment]
@natolambert because it created an error with installing flash-attention: https://github.com/Dao-AILab/flash-attention/issues/453#issuecomment-2129831148

this is why, I added setuptools<70.0.0: https://github.com/Dao-AILab/flash-attention/issues/453#issuecomment-2129831148

File: Dockerfile [hunk 1]
```diff
@@ -113,12 +113,13 @@ RUN apt update && apt install -y openjdk-8-jre-headless
...
 RUN curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
 RUN apt-get -y install git-lfs
+RUN apt-get install -y --reinstall python-pkg-resources
 
 WORKDIR /stage/
 
 # TODO When updating flash-attn or torch in the future, make sure to update the version in the requirements.txt file. 
 COPY requirements.txt .
-RUN pip install --upgrade pip setuptools wheel
+RUN pip install --upgrade pip "setuptools<70.0.0" wheel
```
Comment (natolambert) [inline Dockerfile]:
```suggestion
RUN pip install --upgrade pip "setuptools<70.0.0" wheel 
# TODO, unpin setuptools when this issue in flash attention is resolved
```

Comment (natolambert) [reply Dockerfile]: Merging this.

**Review by natolambert — APPROVED** [review]

**Pull request merged**